clusterloader/README.md

```diff
@@ -0,0 +1,78 @@
+# Cluster Loader
+
+Cluster Loader is a tool we use to deploy large numbers of various Kubernetes objects to a cluster. Currently, Cluster Loader has only been tested against local clusters.
```

Comment:
It's probably worth making the "Currently Cluster Loader has only been tested..." a subtitle or sth (it'll be easier to remove later on)